noetic/distribution.yaml

```diff
@@ -11664,21 +11664,6 @@ repositories:
       url: https://github.com/mikeferguson/simple_grasping.git
       version: ros1
     status: maintained
-  skyway:
-    doc:
-      type: git
-      url: https://github.com/ntt-t3/skyway_for_ros.git
-      version: main
-    release:
```

Comment:
Please remove only the release stanza, and keep the `doc` and `source` stanza. That way your package will be listed on index.ros.org, but won't be built by the buildfarm.